terraform/destroy.sh

```diff
@@ -20,7 +20,7 @@ set -o errexit  # Exit on error
 #set -x
 
 log() { echo "$1" >&2;  }
-
+echo 'Start shutting down resources and deleting the project, this should take about 20 mins. Please do not close the session.'
```

Comment:
The time yes is correct, I tested it a few dozen times and it took me between 10-22˜ minutes.

I can change to up to 20 minutes.
Regarding the text, yes the deletion start only after choosing the project, but if we are not promoting any message here from user percepite it feel like nothing happen. 
Do you have a suggestion for alternative text ? e.g.  
````
You will shortly be promoted to choose a project to delete and shutting down associated resources. This process can take up to 20 mins, please do not close the session in the meantime.
```

Reply:
Marking the project for deletion should be a simple API call, we shouldn't expect a 20 minute wait for that.

I think the problem here is that the project look-up query before the deletion is now stalling. We should look into how to speed up that part of the code and fix that first. Let me know if you want to take that on, otherwise I can take a look

Reply:
Actually I did some testing, and it looks like the problem is still related to sandboxctl interactivity. It seems to load the list  after 1 minute with `terraform/destroy.sh`, but sandboxctl hangs when it gets to the `select` line

Reply:
I checked again, as you stated list takes about a minute and it's mainly do to the loop. I couldn't find a better way to find "owned projects". 
 `gcloud projects delete $PROJECT_ID` took about 12 minutes from the script , email was sent immediately and outside of script also got immediate response.  It seems its not due to the look-up before, but due to `gcloud projects list ` after project deletion.
 Instead if we use `gcloud projects describe cloud-ops-sandbox-3270241197 --flatten="lifecycleState"  |grep "ACTIVE"` it will speed things up (used grep as filter nor format dont seems to work well with describe, if you find notehr hack let me know)

Reply:
Please see the update. 
- I still recommend though to add the message before project list as it can take time depends on permissions and it can be confusing to the user.
- Also do we still to filter the old format `id:stackdriver-sandbox-* AND name='Stackdriver Sandbox Demo'`  ?